ui/src/datasource/datasourceRest.ts

```diff
@@ -27,24 +27,34 @@ const reviver = (key: string, value: object): object => {
 }
 
 export async function getAllDatasources (): Promise<Datasource[]> {
-  const response = await http.get('/')
+  const response = await http.get('/datasources')
   return JSON.parse(response.data, reviver)
 }
 
 export async function getDatasourceById (id: number): Promise<Datasource> {
-  const response = await http.get(`/${id}`)
+  const response = await http.get(`/datasources/${id}`)
   return JSON.parse(response.data, reviver)
 }
 
 export async function createDatasource (datasource: Datasource): Promise<Datasource> {
-  const response = await http.post('/', JSON.stringify(datasource))
+  const response = await http.post('/datasources', JSON.stringify(datasource))
   return JSON.parse(response.data, reviver)
 }
 
 export async function updateDatasource (datasource: Datasource): Promise<AxiosResponse> {
-  return http.put(`/${datasource.id}`, JSON.stringify(datasource))
+  return http.put(`/datasources/${datasource.id}`, JSON.stringify(datasource))
 }
 
 export async function deleteDatasource (id: number): Promise<AxiosResponse> {
-  return http.delete(`/${id}`)
+  return http.delete(`/datasources/${id}`)
+}
+
+export async function getDatasourceData (id: number): Promise<any> {
+  const importResponse = await http.post<string>(`/datasources/${id}/trigger`)
```

Comment:
For the future, we will want to just take the last already imported data instead of triggering it over and over again (TODO: link issue here)

Reply:
#135